packages/sanity-web/src/utils/urls/index.ts

```diff
@@ -114,3 +115,24 @@ export function stringToPathname(
 export function slugify(input: string) {
   return speakingurl(input);
 }
+
+export function isExternalUrl(url: string) {
+  const regex =
+    /^((http|https):\/\/)?[a-zA-Z0-9]+([-\.]{1}[a-zA-Z0-9]+)*\.[a-zA-Z0-9-]{2,}(:[0-9]{1,5})?(\/.*)?$/;
+  return regex.test(url);
+}
+
+export function pathToAbsUrl(args: {
+  path: Slug | null | string;
```

Comment:
I wonder if path should just be a string and before calling it, we should make sure to pass a string value by checking for null or grabbing slug.current? Make the function cleaner.

Reply:
Done